http-api/src/main.rs

```diff
@@ -51,13 +73,44 @@ impl From<Options> for api::Options {
     }
 }
 
+fn set_up_plain_log_format(_opts: &Options) {
+    tracing_subscriber::fmt()
+        .with_env_filter(
+            tracing_subscriber::EnvFilter::try_from_default_env()
+                .unwrap_or(tracing_subscriber::EnvFilter::new("info")),
+        )
+        .init();
+}
+
+#[cfg(not(feature = "gcp"))]
+fn init_logger(opts: &Options) {
+    set_up_plain_log_format(opts)
+}
+
+#[cfg(feature = "gcp")]
+fn init_logger(opts: &Options) {
+    match opts.log_format {
+        LogFmt::Plain => set_up_plain_log_format(opts),
+        LogFmt::Gcp => {
+            use tracing_stackdriver::Stackdriver;
+            use tracing_subscriber::Layer;
+            use tracing_subscriber::{layer::SubscriberExt, Registry};
+            let env_filter = tracing_subscriber::EnvFilter::try_from_default_env()
+                .unwrap_or(tracing_subscriber::EnvFilter::new("info"));
+            let stackdriver_layer = Stackdriver::default();
+            let subscriber = Registry::default().with(stackdriver_layer);
+            let result = env_filter.with_subscriber(subscriber);
+            tracing::subscriber::set_global_default(result)
+                .expect("Could not set up global logger");
+        }
+    }
+}
```

Comment:
Can we create a `logger` module at the top level? This code can then be shared between the two crates.

Reply:
Sure

Reply:
Done